Libraries/LibWeb/WebAudio/ScriptProcessorNode.idl

```diff
@@ -0,0 +1,7 @@
+#import <WebAudio/AudioNode.idl>
+
+[Exposed=Window]
```

Comment:
Could you include a spec link to the interface here please?

The relevant link would be: https://webaudio.github.io/web-audio-api/#ScriptProcessorNode